static/css/components/work.less

```diff
@@ -1,13 +1,26 @@
 @import (less) "base/dl.less";
 
 .workDetails {
-  padding-bottom: 15px;
+  padding: 20px 0;
+  .section {
+    text-align: justify;
+  }
 }
 
 // Left (editionCover)
-div.editionCover {
+.editionCover {
   min-height: 10px;
   padding-right: 1.5em;
+
+  .illustration {
+    transform: perspective(800px) rotateX(10deg);
+    transition: 0.8s;
```

Comment:
Only transition the transform and the opacity; it can cause some ugly perf issues otherwise: `transition: transform 0.8s, opacity 0.8s`